java/ql/src/semmle/code/java/frameworks/apache/Lang.qll

```diff
@@ -0,0 +1,9 @@
+/* Definitions related to the Apache Commons Exec library. */
```

Comment:
Apache Commons Exec?
Is this really correct?

Reply:
Nope, good catch!